dts/bindings/pwm/fsl,imx27-pwm.yaml

```diff
@@ -1,9 +1,9 @@
 # Copyright (c) 2018, Diego Sueiro <diego.sueiro@gmail.com>
 # SPDX-License-Identifier: Apache-2.0
 
-description: i.MX7D PWM
+description: i.MX6SX and i.MX7D PWM
 
-compatible: "fsl,imx7d-pwm"
+compatible: "fsl,imx27-pwm"
```

Comment:
I don't follow the explanation in the commit message for why this is being done. Is the original compatible wrong somehow?

Reply:
Hello, this has been done because the hardware module is the same module present starting from imx27. This driver supports both imx6sx and imx7d and hypothetically other CPU with imx27 module. The same thing was suggested for i2c and I follow the same way with this pr:  https://github.com/zephyrproject-rtos/zephyr/pull/30171#pullrequestreview-535804900 
Thanks for your review

Reply:
Thanks for the pointer! No objections from me if the current NXP maintainers are happy then.

Reply:
Thank you for your changes, a comment in the code about this would be useful.

Reply:
hello @mmahadevan108 i've changed the binding description accordingly: where do you suggest to put comments? Thanks for your review

Reply:
It might be useful to detail why this is called `imx27` in the description.

Reply:
@mmahadevan108 and @dleach02 i've updated the driver description as @mmahadevan108 suggested. 
Thanks for your review